themes/default/content/registry/packages/aws/installation-configuration.md

```diff
@@ -208,3 +208,73 @@ Use `pulumi config set aws:<option>` or pass options to the [constructor of `new
 | `skipRegionValidation` | Optional | Skip static validation of region name. Used by users of alternative AWS-like APIs or users w/ access to regions that are not public (yet). |
 | `skipRequestingAccountId` | Optional | Skip requesting the account ID. Used for AWS API implementations that do not have IAM/STS API and/or metadata API. |
 | `token` | Optional | Use this to set an MFA token. It can also be sourced from the `AWS_SESSION_TOKEN` environment variable. |
+
+## Centralize your configuration
```

Comment:
Since this is an option for configuring AWS, what do you think about listing it under the Configuration section of this page? I ask because the paragraph that introduces that section says, "There are a few different ways you can configure your AWS credentials to work with Pulumi", and this is another one of those ways. What do you think? Any reason not to?

Reply:
I was initially thinking the same thing as far as putting it in that area, but I felt a bit better about it being in its own section given the flow of the content. There's the section right above it that talks about the different configuration options and I like that we can say "hey it's not just the AWS environment variables, you can centralize these other config options we mentioned too". Also I like that users can navigate the different sub sections in this section using the left hand menu which they wouldn't see if it was included further above. I wouldn't say this is a hill I'm willing to die on though lol happy to chat about it

Reply:
Yeah, we should talk through it. In addition to just "keeping all the configuration options together", this approach could also help in preventing ESC from appearing too complicated. Just having more content can make it look like "whoa, I don't want to take all that on", when in fact (especially with AWS) it's actually pretty simple to set up and use. So to the extent we can make this stuff appear as compact and simple as we can, it'll probably help encourage users to give it a try.